pkg/api/http.go

```diff
@@ -33,16 +34,17 @@ const (
 	urlParamSpansPerSpanSet = "spss"
 
 	// backend search (querier/serverless)
-	urlParamStartPage     = "startPage"
-	urlParamPagesToSearch = "pagesToSearch"
-	urlParamBlockID       = "blockID"
-	urlParamEncoding      = "encoding"
-	urlParamIndexPageSize = "indexPageSize"
-	urlParamTotalRecords  = "totalRecords"
-	urlParamDataEncoding  = "dataEncoding"
-	urlParamVersion       = "version"
-	urlParamSize          = "size"
-	urlParamFooterSize    = "footerSize"
+	urlParamStartPage        = "startPage"
+	urlParamPagesToSearch    = "pagesToSearch"
+	urlParamBlockID          = "blockID"
+	urlParamEncoding         = "encoding"
+	urlParamIndexPageSize    = "indexPageSize"
+	urlParamTotalRecords     = "totalRecords"
+	urlParamDataEncoding     = "dataEncoding"
+	urlParamVersion          = "version"
+	urlParamSize             = "size"
+	urlParamFooterSize       = "footerSize"
+	urlParamDedicatedColumns = "dc"
```

Comment:
I understand why this is needed, but I think exposing the parquet config here is indicative of a leaky abstraction.  Not blocking, but tempted to just send the BlockMeta instead of all the individual params.

Reply:
I agree. It would be nice to refactor the way the search block request is built. But I think it's better to do this outside of a larger feature PR.